Add T-lettered article stubs (#4698)
ppy/osu-wiki: wiki/Glossary/Touhosu!/en.md

---
stub: true
tags:
  - feature request

>   - legacy
>   - history

Reviewer: i don think those are needed
```suggestion
```

  - game mode
  - touhou
  - bullet hell
  - dodge the beat
---

# Touhosu!

> For the Touhosu! feature request, see: [Touhosu! / Dodge the Beat](https://osu.ppy.sh/community/forums/topics/19307)*

Reviewer 2: ```suggestion
*For the Touhosu! feature request, see: [Touhosu! / Dodge the Beat](https://osu.ppy.sh/community/forums/topics/19307)*
```

> **Touhosu!** is a requested game mode (proposed in 2011) that proposed osu! include a game mode that combines elements of [osu!catch](/wiki/Game_mode/osu!catch) and the [Touhou Project](https://en.wikipedia.org/wiki/Touhou_Project "Wikipedia") (a bullet hell shoot 'em up video game series). This mode was inspired by the aforementioned Touhou Project, as well as a previous feature request called "Dodge the Beat," that aimed to create a game mode in which players dodge fruits, as opposed to catching them like in osu!catch.

Reviewer: ```suggestion
**Touhosu!** is a feature request from 2011 that proposed a new game mode that combines elements of [osu!catch](/wiki/Game_mode/osu!catch) and the [Touhou Project](https://en.wikipedia.org/wiki/Touhou_Project "Wikipedia") (a bullet hell shoot 'em up video game series). This mode was inspired by the aforementioned Touhou Project, as well as a previous feature request called "Dodge the Beat," that aimed to create a game mode in which players dodge fruits, as opposed to catching them like in osu!catch.
```

> The feature request garnered significant attention from users, with over 522 total replies. However, as of right now, there are reportedly no plans to add Touhosu! or Dodge the Beat into osu!.

Reviewer: ```suggestion
The feature request garnered significant attention from users, with over 522 total replies. However, as for now, there are reportedly no plans of adding Touhosu! or Dodge the Beat into osu!.
```
